<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="local" default="false" doc="true: laptop. false: vehicle."/>
    <arg name="pkg_name" value="vehicle_detection"/>
    <arg name="node_name" default="vehicle_filter_node"/>

    <group ns="$(arg veh)">        
        <node if="$(arg local)" name="vehicle_filter_node" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen"/>
        <include unless="$(arg local)" file="$(find duckietown)/machines"/>
        <node unless="$(arg local)" machine="$(arg veh)" name="virtual_mirror_node" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" />
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

</launch>

